libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -1221,7 +1220,12 @@ void AC_AutoTune_Multi::twitch_test_init()
     } else {
         rotation_rate_filt.reset(0);
     }
-
+    twitch_first_iter = true;
+    test_rate_max = 0.0;
+    test_rate_min = 0.0;
+    test_angle_max = 0.0;
+    test_angle_min = 0.0;
+    rotation_rate_filt.reset(0.0);
```

Comment:
Hi @lthall,

I think there may be an issue here with the rotation_rate_filt.reset(0.0) call because it is called juste a few lines higher up where it may set the rate to the "start_rate".  That line has no effect with this change

Reply:
Thanks, fixed.